src/main/resources/db/changelog/changes/003-create-roles-table.yaml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-roles-table
+      author: mateacademystudent
+      changes:
+        - createTable:
+            tableName: roles
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
```

Comment:
It looks strange that you created a table with the wrong column name and renamed it in one PR. Can you create a table with the correct name?